### PR TITLE
Release v0.6.0-beta1

### LINKED ISSUE #67: Tag v0.6.0-beta1

### PR DESCRIPTION
**Pull request by maurolacy**
Closes #67.

For reference, unified diff of schemas between v0.5.5 and this version:
[schema_diffs-v0.5.5-v0.6.0-beta1.txt](https://github.com/confio/poe-contracts/files/7994478/schema_diffs-v0.5.5-v0.6.0-beta1.txt)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,9 +1,69 @@
 # Changelog
 
-
 ## [Unreleased](https://github.com/confio/poe-contracts/tree/HEAD)
 
-[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.5.4...HEAD)
+[Full Changelog](https://github.com/confio/poe-contracts/compare/v0.6.0-beta1...HEAD)
+
+## [v0.6.0-alpha1](https://github.com/confio/poe-contracts/tree/v0.6.0-beta1) (2022-02-03)
```
Comment (uint) [inline CHANGELOG.md]:
```suggestion
## [v0.6.0-beta1](https://github.com/confio/poe-contracts/tree/v0.6.0-beta1) (2022-02-03)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,9 +1,69 @@
...
+- Validator Voting: check `migrate_msg` content to be \>0 [\#46](https://github.com/confio/poe-contracts/issues/46)
+- Add Open Text Proposals [\#15](https://github.com/confio/poe-contracts/issues/15)
+
+**Merged pull requests:**
+
+- Update rust toolchain to v1.58.1 [\#58](https://github.com/confio/poe-contracts/pull/58) ([uint](https://github.com/uint))
+- 0.6.0-alpha1 release [\#56](https://github.com/confio/poe-contracts/pull/56) ([uint](https://github.com/uint))
+- Voting contract: save info about creator of proposal [\#54](https://github.com/confio/poe-contracts/pull/54) ([ueco-jb](https://github.com/ueco-jb))
+- Fix `remove_hook` helper [\#53](https://github.com/confio/poe-contracts/pull/53) ([maurolacy](https://github.com/maurolacy))
+- Validator set query pagination [\#51](https://github.com/confio/poe-contracts/pull/51) ([maurolacy](https://github.com/maurolacy))
```
Comment (ueco-jb) [inline CHANGELOG.md:44]: Are we backward-nitpicky?:)
This is breaking.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,9 +1,69 @@
...
+- ValidatorVoting - make sure proposal migrate msg is not empty [\#48](https://github.com/confio/poe-contracts/pull/48) ([ueco-jb](https://github.com/ueco-jb))
+- Fix tg4-engagement docs / comments [\#47](https://github.com/confio/poe-contracts/pull/47) ([maurolacy](https://github.com/maurolacy))
+- valset: Add a feature to update `min_weight` and `max_validators` [\#45](https://github.com/confio/poe-contracts/pull/45) ([uint](https://github.com/uint))
+- Update rust to v1.54.0 in CI [\#43](https://github.com/confio/poe-contracts/pull/43) ([maurolacy](https://github.com/maurolacy))
+- Valset: Fix JailMsg inconsistencies [\#39](https://github.com/confio/poe-contracts/pull/39) ([ueco-jb](https://github.com/ueco-jb))
+- Valset: better unjail error message when jail lock didn't expire [\#38](https://github.com/confio/poe-contracts/pull/38) ([ueco-jb](https://github.com/ueco-jb))
+- Store information about operators' validator status [\#37](https://github.com/confio/poe-contracts/pull/37) ([uint](https://github.com/uint))
```
Comment (uint) [inline CHANGELOG.md:51]: This is the one that broke our testnet. I figure it should be in the breaking changes too.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,9 +1,69 @@
...
+**Merged pull requests:**
+
+- Update rust toolchain to v1.58.1 [\#58](https://github.com/confio/poe-contracts/pull/58) ([uint](https://github.com/uint))
+- 0.6.0-alpha1 release [\#56](https://github.com/confio/poe-contracts/pull/56) ([uint](https://github.com/uint))
+- Voting contract: save info about creator of proposal [\#54](https://github.com/confio/poe-contracts/pull/54) ([ueco-jb](https://github.com/ueco-jb))
+- Fix `remove_hook` helper [\#53](https://github.com/confio/poe-contracts/pull/53) ([maurolacy](https://github.com/maurolacy))
+- Validator set query pagination [\#51](https://github.com/confio/poe-contracts/pull/51) ([maurolacy](https://github.com/maurolacy))
+- ValidatorVoting - make sure proposal migrate msg is not empty [\#48](https://github.com/confio/poe-contracts/pull/48) ([ueco-jb](https://github.com/ueco-jb))
+- Fix tg4-engagement docs / comments [\#47](https://github.com/confio/poe-contracts/pull/47) ([maurolacy](https://github.com/maurolacy))
+- valset: Add a feature to update `min_weight` and `max_validators` [\#45](https://github.com/confio/poe-contracts/pull/45) ([uint](https://github.com/uint))
```
Comment (ueco-jb) [inline CHANGELOG.md:47]: This add news endpoint as well.

**Review by ueco-jb — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (uint) [reply CHANGELOG.md]: Ah, done already :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (maurolacy) [reply CHANGELOG.md:44]: Good point. If we all go and add `breaking` to our breaking issues in that list, I can easily regenerate the CHANGELOG.

File: CHANGELOG.md (hunk 3 above)
Comment (maurolacy) [reply CHANGELOG.md:51]: Add the breaking label to the issue, and I'll regenerate the CHANGELOG file.

Comment (maurolacy) [reply CHANGELOG.md]: Never mind, let's (try to) do that from now on. I just moved the identified issues to the "Breaking changes" section.

**Review by ethanfrey — COMMENTED** [review]
Nice to see the breaking changes listed

**Pull request merged**